x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_enrichment/utils.ts

```diff
@@ -150,6 +174,10 @@ export const convertFormStateToProcessor = (formState: ProcessorFormState): Proc
     };
   }
 
+  if (configDrivenProcessors[formState.type]) {
+    return configDrivenProcessors[formState.type].convertFormStateToConfig(formState as any);
```

Comment:
Same as above, why do we need this assertion to any?

Reply:
Same reason for this one: https://github.com/elastic/kibana/pull/216168#discussion_r2022690793